doc/sphinx/source/community/contact.rst

```diff
@@ -0,0 +1,29 @@
+.. _contact:
+
+Contact information
+===================
+
+Core development team
+---------------------
+
+-  Deutsches Zentrum für Luft- und Raumfahrt (DLR), Institut für Physik
+   der Atmosphäre, Germany (PI)
+
+   ESMValTool Core PI and Developer: contact for requests to use the
```

Comment:
I do not understand this sentence. This perhaps needs re-doing after the new goverance. This section also does not contain any contact info ;-)

Reply:
Yes, I'll make an issue. I tried not to work too much on the content in this pull request, I mostly shuffled existing content around.